blogs/2025/02/24/introducing-copilot-agent-mode.md

```diff
@@ -0,0 +1,118 @@
+---
+Order: 93
+TOCTitle: Copilot Agent Mode (preview)
+PageTitle: Copilot Agent Mode (preview)
+MetaDescription: Announcing the GitHub Copilot agent mode in Visual Studio Code.
+Date: 2024-02-24
```

Comment:
There's a typo here, it should be 2025 (it showed up as unread in my RSS feed reader 🙃).

Reply:
Thank you. Will fix now